roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -57,20 +57,20 @@
 
 - name: Add statusDB envvar to sourceme 
   lineinfile: dest={{ ngi_pipeline_conf }}/{{ bash_env_sthlm_script }}
-              line='export STATUS_DB_CONFIG={{ ngi_pipeline_conf }}/statusdb.yaml'
+              line="export STATUS_DB_CONFIG={{ ngi_pipeline_conf }}/{{ statusdb_creds }}"
               backup=no
   tags: ngi_reports
 
-###END STOCKHOLM SPECIFIX
+###END STOCKHOLM SPECIFIC
 
 #NOTE: WORKAROUND MAY NOT BE REQUIRED, BUT LIKELY SO.
 - name: Copy Pandoc fix
-  copy: src=make_report.sh dest="{{ ngi_resources }}/make_report.sh"
+  copy: src=make_report.sh dest="{{ ngi_resources }}/make_report.sh" mode=0770
```

Comment:
Is `0770` really correct? That will set `u=rwx,g=rwx,o=`, i.e. no one outside of the `ngi-sw` group will be able to read/execute. I'm guessing you want `0775`. Or just use a `mode=a+x` if you know that the file has correct read/write permissions in the repo.